server/src/main/resources/logback.xml

```diff
@@ -20,7 +20,7 @@
     <appender name="CandlepinAppender" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${LOG_DIR}/candlepin.log</file>
         <encoder>
-            <pattern>%d{ISO8601} [thread=%thread] [%X{requestType}=%X{requestUuid}, org=%X{org}, csid=%X{csid}] %-5p %c - %m%n</pattern>
+            <pattern>%d{ISO8601} [thread=%thread] [%X{requestType}=%X{requestUuid}, %replace(job_key=%X{jobKey}, ){'job_key=, ', ''} org=%X{org}, csid=%X{csid}] %-5p %c - %m%n</pattern>
```

Comment:
No space between the final ```}``` and the ```org=```. If we have it in there, you get a double space between org and the preceding value.

Reply:
fixed.